test/run_test.py

```diff
@@ -135,14 +138,20 @@ def run_test(executable, test_module, test_directory, options, *extra_unittest_a
     # Can't call `python -m unittest test_*` here because it doesn't run code
     # in `if __name__ == '__main__': `. So call `python test_*.py` instead.
     argv = [test_module + '.py'] + unittest_args + list(extra_unittest_args)
-
     command = executable + argv
     return shell(command, test_directory)
 
 
 def test_cuda_primary_ctx(executable, test_module, test_directory, options):
     return run_test(executable, test_module, test_directory, options, '--subprocess')
 
+def test_jit_ge(executable, test_module, test_directory, options):
+    tokens = test_module.split('_')
```

Comment:
this is literally to tell if we are running test_jit or test_jit_fuser. Frankly, might make more sense to just special case :-) or just use a switch.